src/universal/.devcontainer/devcontainer.json

```
@@ -9,10 +9,9 @@
             "userUid": "1000",
             "userGid": "1000"
         },
-        "ghcr.io/devcontainers/features/dotnet:1": {
-            "version": "7.0.306",
-            "installUsingApt": "false",
-            "additionalVersions": "6"
+        "ghcr.io/devcontainers/features/dotnet:2": {
```

```suggestion
        "ghcr.io/devcontainers/features/dotnet:1": {
```

Can you apply this one change and see if it fixes oryx build issues? We did major refactoring to the .Net Feature which also looks suspicious. This could also explain why oryx does not use .Net 7 from apt and prefers the one installed via Feature (I could be wrong as well)

> Update .NET feature to version 2

If v1 of the dotnet Feature helps us get this PR unblocked then we could work on this ^ in a separate PR/issue.

---

Switching back to `1` did not resolve the Oryx build issues. I switched back to `2` and then added .NET 7.